L1Trigger/L1TGlobal/src/AXOL1TLTemplate.cc

```diff
@@ -0,0 +1,75 @@
+// this class header
+#include "L1Trigger/L1TGlobal/interface/AXOL1TLTemplate.h"
+
+// system include files
+#include <iostream>
+#include <iomanip>
+
+AXOL1TLTemplate::AXOL1TLTemplate() : GlobalCondition() { m_condCategory = l1t::CondAXOL1TL; }
+
+AXOL1TLTemplate::AXOL1TLTemplate(const std::string& cName) : GlobalCondition(cName) {
+  m_condCategory = l1t::CondAXOL1TL;
+}
+
+AXOL1TLTemplate::AXOL1TLTemplate(const std::string& cName, const l1t::GtConditionType& cType)  //not sure we need cType
+    : GlobalCondition(cName, l1t::CondAXOL1TL, cType) {
+  int nObjects = nrObjects();
+
+  if (nObjects > 0) {
+    m_objectType.reserve(nObjects);
+  }
+}
+
+// copy constructor
+AXOL1TLTemplate::AXOL1TLTemplate(const AXOL1TLTemplate& cp) : GlobalCondition(cp.m_condName) { copy(cp); }
+
+// destructor
+AXOL1TLTemplate::~AXOL1TLTemplate() {
+  // empty now
+}
+
+// assign operator
+AXOL1TLTemplate& AXOL1TLTemplate::operator=(const AXOL1TLTemplate& cp) {
+  copy(cp);
+  return *this;
+}
+
+// setConditionParameter - set the parameters of the condition
+void AXOL1TLTemplate::setConditionParameter(const std::vector<ObjectParameter>& objParameter) {
+  m_objectParameter = objParameter;
+}
```

Comment:
Please move the single line set to the header

Reply:
If this structure is default, it is fine to keep it this way to keep consistent with other conditions. At least as far as I am concerned.